### PR TITLE
Create PR template

### PR DESCRIPTION
**Pull request by piyalibanerjee**

### CODE REVIEW

**Review by christianang — APPROVED** [review]
LGTM.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## All Submissions:
+
+* [ ] Have you followed the guidelines in our Contributing document?
```
Comment (bwinter) [inline .github/pull_request_template.md:3]: Could maybe make this a link to that document?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/pivotal/cf-onboarding/pulls) for the same update/change?
```
Comment (bwinter) [inline .github/pull_request_template.md:4]: Does it make sense to word these as:
```md
* [ ] I ensured there weren't other open pull requests for the same change?
```
That way when you "check" the box you're kind of ... certifying your actions. 🤔

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+
+## New Stories:
+* [ ] Have you checked to ensure the new stories don't significantly overlap with other stories in the backlog?
```
Comment (bwinter) [inline .github/pull_request_template.md:8]: Might be good to link to the backlog so it's easier for them to check?

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+* [ ] Have you applied the appropriate label for the stories?
```
Comment (bwinter) [inline .github/pull_request_template.md:9]: I wonder how they will know what an appropriate label is? Could we give an example or is there an easy way to see what labels are available? (if so a link could be good)

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+1. How will the new stories enrich the CF-onboarding curriculum?
+2. Brief description of the stories
+
+## Updates to Existing Stories:
```
Comment (bwinter) [inline .github/pull_request_template.md:14]: Are theses basically templates for the brief descriptions? (I am a little confused about this sections purpose.)

**Review by bwinter — COMMENTED** [review]
Thanks for adding this! I added some ideas, feel free to take or leave em!

**Pull request merged**